test/functional/fixtures/regression/gh-2568/test.js

```diff
@@ -0,0 +1,312 @@
+                `);
+            });
+    });
+
+    it('sibling', function () {
+        return runTests('testcafe-fixtures/index.js', 'sibling', { selectorTimeout: 100, shouldFail: true })
+            .catch(function (errs) {
+                assertSelectorCallstack(errs[0], `
+                    The specified selector does not match any element in the DOM tree.
+                        | Selector('body')
+                        |   .find('div.parent > div:first-child > div')
+                      > |   .sibling()
+                        |   .parent('div')
+                        |   .child('span')
+                `);
+            });
+    });
+
+    it('find', function () {
+        return runTests('testcafe-fixtures/index.js', 'find', { selectorTimeout: 100, shouldFail: true })
+            .catch(function (errs) {
+                assertSelectorCallstack(errs[0], `
+                    The specified selector does not match any element in the DOM tree.
+                        | Selector('body')
+                      > |   .find('div.not-existing')
+                        |   .nextSibling()
+                        |   .parent('div')
+                        |   .child('span')
+                `);
+            });
+    });
+
+    it('drag', function () {
+        return runTests('testcafe-fixtures/index.js', 'drag', { selectorTimeout: 100, shouldFail: true })
+            .catch(function (errs) {
+                assertSelectorCallstack(errs[0], `
+                    The specified "destinationSelector" does not match any element in the DOM tree.
+                        | Selector('div.parent')
+                      > |   .child('ul')
+                `);
+            });
+    });
+
+    it('snapshot', function () {
+        return runTests('testcafe-fixtures/index.js', 'snapshot', { selectorTimeout: 100, shouldFail: true })
+            .catch(function (errs) {
+                assertSelectorCallstack(errs[0], `
+                    Cannot obtain information about the node because the specified selector does not match any node in the DOM tree.
+                      > | Selector('ul li')
+                        |   .filter('test')
+                `);
+            });
+    });
+
+    it('custom DOM properties', function () {
+        return runTests('testcafe-fixtures/index.js', 'custom DOM properties', { selectorTimeout: 100, shouldFail: true })
+            .catch(function (errs) {
+                assertSelectorCallstack(errs[0], `
+                    Cannot obtain information about the node because the specified selector does not match any node in the DOM tree.
+                      > | Selector('ul li')
+                `);
+            });
+    });
+
+    it('long selector', function () {
+        return runTests('testcafe-fixtures/index.js', 'long selector', { selectorTimeout: 100, shouldFail: true })
+            .catch(function (errs) {
+                expect(/Selector\('a+\.\.\.\)/.test(errs[0])).to.be.true;
```

Comment:
This test covers only ellipsis. You can convert this functional test to unit.

Reply:
we have special tests for formatting